libmscore/layout.cpp

```diff
@@ -2231,11 +2231,13 @@ void Score::createBeams(Measure* measure)
                         Beam::Mode mode = cr->beamMode();
                         if (mode == Beam::Mode::MID || mode == Beam::Mode::END) {
                               ChordRest* prevCR = findCR(measure->tick() - 1, track);
-                              const Measure* pm = prevCR->measure();
-                              if (prevCR && !beamNoContinue(prevCR->beamMode())
-                                 && !pm->lineBreak() && !pm->pageBreak() && !pm->sectionBreak()) {
-                                    beam = prevCR->beam();
-                                    a1 = beam ? beam->elements().front() : prevCR;
+                              if (prevCR) {
```

Comment:
I imagine this doesn't fix the issue where prevCR ends up being the last chordrest of the piece if we're actually dealing with the first note of the piece?  That's because findCR(-1) ends up returning the last cr of the piece rather than nothing as I think it probably should.  I'd be afraid to 'fix" findCR() in case that is relied on elsewhere, but we might want to check that pm really *is* a previous measure.

Reply:
How?

Reply:
Not sure, maybe by checking its tick?  For that matter it cod be worth checking to be sure it is immediately preceding, not say 4 measures earlier.  I've seen people make that mistake and end up with a beam across the page.  Probably there is no legitimate use case for that.

Reply:
Actually it is `tick2measureMM(-1)` that returns the last measure (and it gets called by `findCR()`